dtests/src/test/java/io/snappydata/hydra/cluster/SnappyTest.java

```diff
@@ -0,0 +1,1097 @@
+
+        snc.sql("CREATE TABLE " + stagingColumnTable + " \n" +
+                " USING parquet OPTIONS(path '" + hfile + "')");
+        Log.getLogWriter().info("SS - staging column table " + stagingColumnTable + " created successfully");
+        snc.sql("CREATE TABLE " + columnTable + " USING column OPTIONS(buckets '11') AS (\n" +
+                "  SELECT Year AS yeari, Month AS monthi , DayOfMonth,\n" +
+                "    DayOfWeek, DepTime, CRSDepTime, ArrTime, CRSArrTime,\n" +
+                "    UniqueCarrier, FlightNum, TailNum, ActualElapsedTime,\n" +
+                "    CRSElapsedTime, AirTime, ArrDelay, DepDelay, Origin,\n" +
+                "    Dest, Distance, TaxiIn, TaxiOut, Cancelled, CancellationCode,\n" +
+                "    Diverted, CarrierDelay, WeatherDelay, NASDelay, SecurityDelay,\n" +
+                "    LateAircraftDelay, ArrDelaySlot\n" +
+                "    FROM " + stagingColumnTable + ")");
+        Log.getLogWriter().info("SS - column table " + columnTable + " created successfully");
+        snc.sql("CREATE TABLE " + stagingRowTable + " \n" +
+                "  USING parquet OPTIONS(path '" + reffile + "')");
+        Log.getLogWriter().info("SS - staging row table " + stagingRowTable + " created successfully");
+        snc.sql("CREATE TABLE " + rowTable + " USING row OPTIONS() AS (SELECT CODE, DESCRIPTION FROM " + stagingRowTable + ")");
+        Log.getLogWriter().info("SS - row table " + rowTable + " created successfully");
+    }
+
+
+    public static void HydraTask_createRowColumnTableUsingDataFrame() {
+        HydraTask_initializeSnappyTest();
+        String hfile = snappyTest.getDataFrameDataLocation() + "/2015.parquet";
+        String reffile = snappyTest.getDataLocation() + "/airportcodeParquetData";
+        DataFrame airlineDataFrame, airlineRefDataFrame;
+        HashMap<String, String> props = new HashMap<String, String>();
+//        props.put("buckets","11");
+        airlineDataFrame = snc.read().load(hfile);
+        snc.read().load(hfile).registerTempTable(stagingColumnTable);
+        Log.getLogWriter().info("SS - airline dataframe loaded successfully");
+        snc.dropTable(columnTable, true);
+        snc.createTable(columnTable, "column", airlineDataFrame.schema(), toScalaMap(props));
+        airlineDataFrame.write().format("column").mode(SaveMode.Append).saveAsTable(columnTable);
+        Log.getLogWriter().info("SS - column table " + columnTable + " created successfully");
+        airlineRefDataFrame = snc.read().load(reffile);
+        snc.read().load(reffile).registerTempTable(stagingRowTable);
+        props.clear();
+        Log.getLogWriter().info("SS - airlineref dataframe loaded successfully");
+        snc.dropTable(rowTable, true);
+        snc.createTable(rowTable, "row", airlineRefDataFrame.schema(), toScalaMap(props));
+        airlineRefDataFrame.write().format("row").mode(SaveMode.Append).saveAsTable(rowTable);
```

Comment:
read comments above regarding the configuring the columntable and rowtable properties trough the conf file